rules/nette/tests/Rector/ClassMethod/RemoveParentAndNameFromComponentConstructorRector/Fixture/control.php.inc

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Nette\ComponentModel;
+
+interface IContainer {}
+
+?>
+
+<?php
+
+namespace Nette\Application\UI;
+
+use Nette\ComponentModel\IContainer;
+
+namespace Rector\Nette\Tests\Rector\ClassMethod\RemoveParentAndNameFromComponentConstructorRector\Fixture;
+
+abstract class Control implements IContainer
+{
+}
+
+?>
+
+<?php
+
+namespace MyNamespace;
+
+use Nette\Application\UI\Control;
+
+namespace Rector\Nette\Tests\Rector\ClassMethod\RemoveParentAndNameFromComponentConstructorRector\Fixture;
+
+class MyComponent extends Control
+{
+    private $parent;
+    
+    private $name;
+    
+    private $value;
+    
+    public function __construct(IContainer $parent = null, $name = null, int $value)
+    {
+        parent::__construct($parent, $name);
+        $this->parent = $parent;
+        $this->name = $name;
+        $this->value = $value;
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Nette\Tests\Rector\ClassMethod\RemoveParentAndNameFromComponentConstructorRector\Fixture;
+
+// what is expected code?
+// should remain the same? delete part bellow ----- (included)
```

Comment:
This section is missing. The expected output must be here

Reply:
Yeah, I have to check how to do it O:-)
But there is another problem:
```
PHP Fatal error:  Cannot declare interface Nette\ComponentModel\IContainer, because the name is already in use in /tmp/_temp_fixture_easy_testing/input_bb1a98f5d08ae38bde39_control.php on line 5
```

Reply:
The class is duplicated :) It's already in the stubs: https://github.com/rectorphp/rector/blob/master/stubs/Nette/ComponentModel/IContainer.php

Reply:
> Yeah, I have to check how to do it O:-) 

You can modify [it here](https://github.com/lulco/rector/edit/patch-1/rules/nette/tests/Rector/ClassMethod/RemoveParentAndNameFromComponentConstructorRector/Fixture/control.php.inc?pr=%2Frectorphp%2Frector%2Fpull%2F5052), or better on creating pull-request next time.

Reply:
@TomasVotruba Failing tests added, now build "failed correctly"